### ISSUE TITLE
What is the difference between [inputa, labela] and [inputb, labelb]?

### ISSUE THREAD

**Issue by liqilei** [JWSoh/MZSR#17]
Hi, thanks for your work. 

I wonder what is the difference between `[inputa, labela]` and `[inputb, labelb]`?

**Comment by JWSoh** [comment]
The former ones are for inner loop of meta-training and the latter ones are for the outer loop of meta-training.

It is easy to understand that the former ones are training set and the latter ones are validation set.

**Comment by liqilei** [comment]
> The former ones are for inner loop of meta-training and the latter ones are for the outer loop of meta-training.
> 
> It is easy to understand that the former ones are training set and the latter ones are validation set.

Got it. Thanks a lot.
